### DISCUSSION TITLE
Tombs of Amascut - Invocations Preset

### DISCUSSION THREAD

**Discussion by A-RooK** [runelite/runelite#15560] [Ideas]
I was not sure if someone was already working on this.  However, it would be really great to get a plugin that allows you to store invocation selections and quickly reload them from a menu on the sidebar.

**Comment by BrandonEnglish848** [comment]
You can store up to 5 presets in-game

**Reply by A-RooK** [reply]
Thanks, I posted this request in August of 2022, the week ToA came out, before invo presets were part of the game.
